spec/classes/strongswan_pki_ca_spec.rb

```diff
@@ -6,7 +6,6 @@
       let(:facts) do
         facts
       end
-      let(:pre_condition) { 'include strongswan' }
```

Comment:
It's good to be able to get rid of the need for the user to manually `include strongswan`.